test/jdk/java/nio/channels/vthread/BlockingChannelOps.java

```diff
@@ -207,16 +222,23 @@ public void testSocketChannelWriteInterrupt() throws Exception {
         VThreadRunner.run(() -> {
             try (var connection = new Connection()) {
                 SocketChannel sc = connection.channel1();
-                ScheduledInterrupter.schedule(Thread.currentThread(), DELAY);
+
+                // delayed interrupt of current thread
+                Thread thisThread = Thread.currentThread();
+                runAfterParkedAsync(thisThread::interrupt);
+
                 try {
                     ByteBuffer bb = ByteBuffer.allocate(100*1024);
                     for (;;) {
                         int n = sc.write(bb);
                         assertTrue(n > 0);
                         bb.clear();
                     }
```

Comment:
Same here? Should we add a `fail` if we don't receive a `ClosedByInterruptException`?